src/main/java/uk/ac/bath/se/DungeonController.java

```diff
@@ -535,13 +538,16 @@ public void assignChest() {
     public String giveRandomItem() {
         Random rand = new Random();
         String message = "";
+        int nextInt = rand.nextInt(2);
+        message += "You open the chest to find...";
```

Comment:
Why is this being printed out twice? Also random use of two variables, it looks like this method did not merge properly.